models/articles.model.js

```diff
@@ -81,7 +81,15 @@ exports.checkArticleExists = (article_id) => {
 };
 
 exports.getArticleByIdModel = (article_id) => {
-  let SQL = `SELECT * FROM articles WHERE article_id = $1;`;
+  let SQL = `SELECT articles.article_id, articles.author, title, 
```

Comment:
Doesnt look like you reassign this variable anywhere, so should ideally be `const`